core/services/bulletprooftxmanager/attempts.go

```diff
@@ -0,0 +1,192 @@
+package bulletprooftxmanager
+
+import (
+	"bytes"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/pkg/errors"
+
+	"github.com/smartcontractkit/chainlink/core/services/gas"
+	"github.com/smartcontractkit/chainlink/core/utils"
+)
+
+var Max256BitUInt *big.Int
+
+func init() {
+	var i, e = big.NewInt(2), big.NewInt(256)
+	Max256BitUInt = i.Exp(i, e, nil)
+}
+
+func NewDynamicFeeAttempt(cfg Config, ks KeyStore, chainID *big.Int, etx EthTx, fee gas.DynamicFee, gasLimit uint64) (attempt EthTxAttempt, err error) {
+	if err = validateDynamicFeeGas(cfg, fee, gasLimit, etx); err != nil {
+		return attempt, errors.Wrap(err, "error validating gas")
+	}
+
+	var al types.AccessList
+	if etx.AccessList.Valid {
+		al = etx.AccessList.AccessList
```

Comment:
There is no followup card, but it would be trivial to enable support for it after this PR. What job types do you think might benefit?

Reply:
I know that VRF jobs would because of how the contract is set up. I suspect basically all jobs could benefit, but how exactly I'd have to read the contract code more carefully. Probably needs a spike first

Reply:
It's basically trivial to specify an AccessList at insert-time. We'd have to take it on a job-by-job basis

Reply:
Created this story to track: https://app.shortcut.com/chainlinklabs/story/17478/use-accesslists-where-appropriate-to-save-gas